tests/other/wasm2c/my-code.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+
+// We could
+//
+//   #include <lib.wasm.h>
+//
+// for the externs declared here manually, but including that currently
+// requires having wasm-rt.h in the include path, which may be annoying for
+// users - needs to be thought about.
+
+extern void wasmbox_init(void);
+
+extern int (*Z_do_bad_thingZ_ii)(int);
+
+extern int (*Z_twiceZ_ii)(int);
```

Comment:
That's name mangling that wasm2c does. cc @binji 

The `Z_` prefix might make it look too much like C++ mangling, which can be confusing. I wonder if it would be `W2C_` or such?